Cargo.toml

```diff
@@ -21,3 +21,10 @@ thiserror = "1.0.40"
 
 [features]
 serde = ["dep:serde", "ordered-float/serde", "indexmap/serde"]
+
+[dev-dependencies]
+serde_json = "1.0.96"
+
+[[test]]
+name = "serde"
+required-features = ["serde"]
```

Comment:
I would prefer not to make serde a default feature because the github workflow runner runs tests with serde enabled and with serde disabled, and this prevents it from running with serde disabled.

Reply:
if you do `cargo test` it simply won't run the serde.rs test, but if you use `cargo test -F serde` it will.

Reply:
Thanks for explaining, this should be fine.